src/classes/VOL_SharedCode_TEST.cls

```diff
@@ -322,6 +322,66 @@ public with sharing class VOL_SharedCode_TEST {
 
     }
 
+    @isTest
+    private static void testCaptchaResponseSuccess() {   
+        VOL_SharedCode.VolunteersSettings.Enable_Site_Captcha__c = true;
+        VOL_SharedCode.VolunteersSettings.Google_reCaptcha_Endpoint__c = 'www.salesforce.com';
+        PageReference lookupPage = Page.PersonalSiteContactLookup;
+        Test.setCurrentPage(lookupPage);
+        System.currentPageReference().getParameters().put('g-recaptcha-response', '1234567890');
+
+        HttpMockFactory mock = new HttpMockFactory(200, 'OK', '{"success":true}', new Map<String,String>());        

```

Comment:
Do we want to worry about handling other http statuses?

Reply:
Right now if the verification returns any status / body value without the success parameter, the verification is treated as a failure and displayed as such.  I will add an additional test that mocks an unexpected response to cover an additional use case.